kpi/signals.py

```diff
@@ -16,6 +18,15 @@
 from kpi.utils.permissions import grant_default_model_level_perms
 
 
+@receiver(config_updated)
+def constance_updated(sender, key, old_value, new_value, **kwargs):
```

Comment:
To be honest, it's a copy-and-paste from the Constance documentation: https://django-constance.readthedocs.io/en/latest/#signals.

What would the advantage be to removing `old_value` and `new_value`? If Constance always sends them, my first thought is that we might as well leave them as named parameters so that some future developer doesn't have to look up the documentation if they need access to the values. I guess you have a different argument, though? :)